components/finance/UnitPriceQuestion.tsx

```diff
@@ -1,19 +1,28 @@
 import React, { useState, useEffect } from "react";
 import { GuessData } from "../../pages/api/guessData";
 import { Button } from "../ui/Button";
+import { name } from "../../pages/api/names";
+import { fruitsMap } from "../../pages/api/WordProblemModelObjects";
 
 export interface UnitPriceQuestionProps {
   total: number;
   numberOfObjects: number;
   submitGuess: (guess: GuessData) => void;
   answer: string;
+  name: string;
 }
 
+const nameSelector = () => {
```

Comment:
Instead of duplicating this function, could you add an export keyword to the existing function and import the function here?

Reply:
Thank you! that makes much more sense on the names....still can't figure out the fruits. it's more involved than what I'm doing I'm sure.
![Screen Shot 2022-08-20 at 10 21 45 PM](https://user-images.githubusercontent.com/111075855/185763274-7999d7b0-6a42-4cad-a5ba-8918ad7bfbf9.png)